providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/JsonBindingProvider.java

```diff
@@ -53,7 +54,9 @@ public Object readFrom(Class<Object> type, Type genericType,
          return jsonb.fromJson(entityStream, genericType);
       } catch (Throwable e)
       {
-         throw new ProcessingException(Messages.MESSAGES.jsonBDeserializationError(e.toString()));
+         LogMessages.LOGGER.debugf(Messages.MESSAGES.jsonBDeserializationError(e.toString()));
```

Comment:
@rsearls  better doing something like:
if (LogMessages.LOGGER.isDebugEnabled()) {
  LogMessages.LOGGER.debugf(Messages.MESSAGES.jsonBDeserializationError(e.toString()));
}
to avoid generating the stacktrace string when not needed